### DISCUSSION TITLE
Please add Nukkit support

### DISCUSSION THREAD

**Discussion by dadencukillia** [minecraft-dev/MinecraftDev#1878] [Ideas]
Hi, I want to create a bedrock server. My kernel is nukkit! Please add Nukkit support...

**Comment by DenWav** [comment]
There's been a PR open for doing this for a while: https://github.com/minecraft-dev/MinecraftDev/pull/1791

Unfortunately we haven't had a chance to look at it yet. I kept meaning too but I'm always getting pulled away, so I apologize for that - and I apologize to @Sparky983 for my inactivity.
